### ISSUE TITLE
Panel intelligent hide

### ISSUE THREAD

**Issue by sadalavoaoj** [linuxmint/Cinnamon#4921]
Hi there,

I have installed cinnamon 17.3, like i had in 17.2 i changed my panel to intelligent hide, but after i start for example spotify and make it run on background the panel doesnt show up again, It stays hide. 

In 17.2 i didnt had that problem, someone with the same issue?

Best regards.

**Comment by DarekDeo** [comment]
Yes and could be related to this issue: https://github.com/linuxmint/Cinnamon/issues/4584

If I remember correctly during situation You mention I had to focus cursor on any other window to be able to see panel. Then fast click on panel few times to make it react.

Additionaly panel intellihide panel does LAG Cinnamon for me a lot, as mentioned here: https://github.com/linuxmint/Cinnamon/issues/2465
Also panel hide makes disappear panel shadow effects: https://github.com/linuxmint/Cinnamon/issues/4503

Issues appear in Linux Mint 17.3 and Cinnamon 2.8. All the issues (I think with exception for window drag lag) appeard in 17.2 too.

Currently I set panel to always show to avoid any problems with it.
